pkg/remote/aws/default_vpc_supplier.go

```diff
@@ -0,0 +1,76 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/aws/aws-sdk-go/service/ec2/ec2iface"
+	"github.com/cloudskiff/driftctl/pkg"
+	"github.com/cloudskiff/driftctl/pkg/remote/deserializer"
+	"github.com/cloudskiff/driftctl/pkg/resource/aws"
+	awsdeserializer "github.com/cloudskiff/driftctl/pkg/resource/aws/deserializer"
+	"github.com/zclconf/go-cty/cty"
+
+	"github.com/cloudskiff/driftctl/pkg/resource"
+	"github.com/cloudskiff/driftctl/pkg/terraform"
+
+	"github.com/sirupsen/logrus"
+)
+
+type DefaultVPCSupplier struct {
+	reader       terraform.ResourceReader
+	deserializer deserializer.CTYDeserializer
+	client       ec2iface.EC2API
+	runner       *terraform.ParallelResourceReader
+}
+
+func NewDefaultVPCSupplier(runner *pkg.ParallelRunner, client ec2iface.EC2API) *DefaultVPCSupplier {
+	return &DefaultVPCSupplier{
+		terraform.Provider(terraform.AWS),
+		awsdeserializer.NewDefaultVPCDeserializer(),
+		client,
+		terraform.NewParallelResourceReader(runner),
+	}
+}
+
+func (s DefaultVPCSupplier) Resources() ([]resource.Resource, error) {
+	input := ec2.DescribeVpcsInput{}
+	var defaultVPCs []*ec2.Vpc
+	err := s.client.DescribeVpcsPages(&input,
+		func(resp *ec2.DescribeVpcsOutput, lastPage bool) bool {
+			for _, vpc := range resp.Vpcs {
+				if vpc.IsDefault != nil && *vpc.IsDefault {
```

Comment:
@moadibfr WDYT of handling both `aws_default_vpc` and `aws_vpc` in a single supplier ? May I rename this supplier to `aws_vpc_supplier.go` and handle both default and non default VPCs ?

It will avoid nearly duplicated logic.

Reply:
I think it's a good idea to avoir duplication as this is almost identical. And yeah `aws_vpc_supplier` might be clearer. We might want to add a little comment on the struct to warn that it supply the two resources also.

Reply:
👍🏻